frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/vm-settings-tab.tsx

```diff
@@ -138,8 +149,10 @@ const stateToProps = (state, { wizardReduxID }) => ({
   commonTemplates: iGetCommonData(state, wizardReduxID, VMWizardProps.commonTemplates),
   userTemplateName: iGetCommonData(state, wizardReduxID, VMWizardProps.userTemplateName),
   userTemplates: iGetCommonData(state, wizardReduxID, VMWizardProps.userTemplates),
+  commonDataVolumes: iGetCommonData(state, wizardReduxID, VMWizardProps.openshiftCNVBaseImages),
```

Comment:
it is a bit misleading to call this datavolumes since they are pvcs. Can we call them cnvBaseImages or something more relatable?

Reply:
done